app/scripts/directives/bindService.js

```diff
@@ -291,5 +294,11 @@
         ctrl.onClose();
       }
     };
+
+    ctrl.bindCreated = function(binding) {
```

Comment:
Do we need to add this to the `ctrl`? Doesn't seem to be used in the view, maybe we can have it in as an internal function instead?

Reply:
It's a fair point. I am following the current style in the file which I would prefer for consistency.